products/visualstudio.md

```diff
@@ -3,60 +3,75 @@ title: Visual Studio
 layout: post
 category: app
 sortReleasesBy: "cycleShortHand"
-
+releaseLabel: '__RELEASE_CYCLE__ __CODENAME__'
 # A list of releases, supported or not
 # Newer releases go on top of the list, in order
 releases:
-  - releaseCycle: "2022 version 17.0"
+  - releaseCycle: "2022"
+    codename: "17.0"
```

Comment:
I wonder if "version" should stay in the name?

Is "2022 version 17.0" more readable than "2022 17.0"?

https://endoflife.date/visualstudio

https://deploy-preview-891--endoflife-date.netlify.app/visualstudio

And maybe part of the official name?

https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes